easybuild/easyconfigs/p/pangolin/pangolin-3.1.10-foss-2021a.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'PythonBundle'
+
+name = 'pangolin'
```

Comment:
```suggestion
name = `Pangolin'
```
In my opinion

Reply:
At their [GitHub](https://github.com/cov-lineages/pangolin) it is actually all in lower cases which I would take as a guide here. 
However at their [webpage](https://cov-lineages.org/resources/pangolin.html) it is capitalized. I could ask my colleague if there is a preference in the community or maybe @SethosII has an opinion here.

Reply:
I would also use the lowercase version. They use the uppercase name only in some places, but the majority has the lowercase version including their [setup.py](https://github.com/cov-lineages/pangolin/blob/master/setup.py#L8) and [environment.yml](https://github.com/cov-lineages/pangolin/blob/master/environment.yml#L1).

Reply:
I have created PR [13733](](https://github.com/easybuilders/easybuild-easyconfigs/pull/13733) for the new version of pangoline